apps/etterlatte-saksbehandling-ui/client/src/components/behandling/BehandlingRoutes.tsx

```diff
@@ -124,20 +125,21 @@ const hentAktuelleRoutes = (behandling: IBehandlingReducer | null) => {
 }
 
 export function revurderingRoutes(behandling: IBehandlingReducer): Array<BehandlingRouteTypes> {
-  const defaultRoutes: Array<BehandlingRouteTypes> = [
-    { path: 'revurderingsoversikt', description: 'Revurderingsoversikt' },
-    {
-      path: 'vilkaarsvurdering',
-      description: 'Vilkårsvurdering',
-      kreverBehandlingsstatus: IBehandlingStatus.VILKAARSVURDERT,
-    },
-    {
-      path: 'beregningsgrunnlag',
-      description: 'Beregningsgrunnlag',
-      kreverBehandlingsstatus: IBehandlingStatus.VILKAARSVURDERT,
-    },
-    { path: 'beregne', description: 'Beregning', kreverBehandlingsstatus: IBehandlingStatus.BEREGNET },
-  ]
+  const revurderingsoversikt = { path: 'revurderingsoversikt', description: 'Revurderingsoversikt' }
+  const vilkaarsvurdering = {
+    path: 'vilkaarsvurdering',
+    description: 'Vilkårsvurdering',
+    kreverBehandlingsstatus: IBehandlingStatus.VILKAARSVURDERT,
+  }
+  const beregningsgrunnlag = {
+    path: 'beregningsgrunnlag',
+    description: 'Beregningsgrunnlag',
+    kreverBehandlingsstatus: IBehandlingStatus.VILKAARSVURDERT,
+  }
+  const beregning = { path: 'beregne', description: 'Beregning', kreverBehandlingsstatus: IBehandlingStatus.BEREGNET }
+  const defaultRoutes: Array<BehandlingRouteTypes> = erOpphoer(behandling.revurderingsaarsak!!)
```

Comment:
Har vi tenkt noe rundt at feks når vi får inn en generell revurderingsmulighet (revurderingsårsak: ANNEN), så vil det være vilkårsvurderingen som avgjør om det er opphør eller ikke. Tilsvarende kan man i en av revurderingene som nå er `true` for `erOpphoer` kunne velge å oppfylle vilkårsvurderingen. Burde det vært sperre for dette på noe nivå?

Reply:
Det korte svaret er nok nei, men det må vi ta høgde for når vi byrjar med _annen_, ja.